lib/Alembic/AbcCoreOgawa/StreamManager.cpp

```diff
@@ -47,7 +47,18 @@ namespace ALEMBIC_VERSION_NS {
 #define COMPARE_EXCHANGE( V, COMP, EXCH ) V.compare_exchange_weak( COMP, EXCH, std::memory_order_seq_cst, std::memory_order_seq_cst )
 // Windows
 #elif defined( _MSC_VER )
-#define COMPARE_EXCHANGE( V, COMP, EXCH ) InterlockedCompareExhange64( &V, EXCH, COMP ) == COMP
+#define COMPARE_EXCHANGE( V, COMP, EXCH ) InterlockedCompareExchange64( &V, EXCH, COMP ) == COMP
+int ffsll(long	long value)
+{
+	if (!value)
+		return 0;
+
+	for (int bit = 0; bit < 63; bit++)
```

Comment:
Yes, it should be 64, _BitScanForward64 is only available on X64, so i opted for a more generic solution that would still work for x86